std/node/util.ts

```diff
@@ -0,0 +1,12 @@
+export {
+  isBoolean,
+  isNull,
+  isNullOrUndefined,
+  isNumber,
+  isString,
+  isSymbol,
+  isUndefined,
+  isObject,
+  isError,
+  isFunction
+} from "./util/main.ts";
```

Comment:
Let's change this from `main.ts` to `mod.ts`.

I am not quite sure if we should reexpose mod as `std/node/util.ts` or just leave as `std/node/util/mod.ts`. The latter follows our recommendation better, while the former might seem more familiar to node users...

Reply:
`mod.ts` seems better yes.
Regarding the exposure it's more like a taste, you have 2 solutions
`import { fs } from 'std/node'` or `import { readfile } from 'std/node/fs'` i'm ok with both